packages/nibijs/src/msg/staking.ts

```diff
@@ -0,0 +1,57 @@
+import { EncodeObject, GeneratedType, Registry } from "@cosmjs/proto-signing";
+import * as pb from "@nibiruchain/protojs/dist/staking/v1/tx"
+import { MsgTypeUrls, TxMessage } from "./types";
+
+export const stakingMsgTypeUrls: MsgTypeUrls = {
+  MsgAddMargin: `/${pb.protobufPackage}.MsgBeginRedelegate`,
+  MsgRemoveMargin: `/${pb.protobufPackage}.MsgCreateValidator`,
+  MsgLiquidate: `/${pb.protobufPackage}.MsgDelegate`,
+  MsgOpenPosition: `/${pb.protobufPackage}.MsgEditValidator`,
+  MsgClosePosition: `/${pb.protobufPackage}.MsgUndelegate`,

```

Comment:
typo here: It still has message names from the `perp` module, but these should match the staking names: `MsgBeginRedelegate`, `MsgDelegate`, etc.